src/http/override.ts

```diff
@@ -49,13 +49,12 @@ function handleRequest(
  */
 export const overrideHttpModule: ModuleOverride = (middleware) => {
   let patchedModules: PatchedModules = {}
-  const modules: ['http', 'https'] = ['http', 'https']
+  const modules = ['http', 'https']
 
   modules.forEach((protocol) => {
-    const module = ({
-      http: require('http'),
-      https: require('https'),
-    } as Record<string, typeof http | typeof https>)[protocol]
+    const module = protocol === 'https'
+      ? require('https') as typeof https
+      : require('http') as typeof http;
```

Comment:
What is the practical benefit of these changes? Improved type coverage of the imported modules?

Reply:
Just cosmetic. Actually, looking at it again i guess it could be just `const module = protocol === 'https' ? https: http`, since they are already imported in the top scope.